airbyte-integrations/connectors/source-partnerstack/source_partnerstack/partnerstack.yaml

```diff
@@ -0,0 +1,98 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["data", "items"]
+  requester:
+    url_base: "https://api.partnerstack.com/api/v2/"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['public_key'] }}"
+      password: "{{ config['private_key'] }}"
+    request_options_provider:
+      request_parameters:
+        min_created: "{{ timestamp(config['start_date']) * 1000 }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      pagination_strategy:
+        type: "CursorPagination"
+        cursor_value: "{{ last_records[-1]['key'] }}"
```

Comment:
Just curious about the usage of this option.

Reply:
Which option are you referring to?

Reply:
the cursor field, this is the first connector using this feature interpolation

Reply:
Not sure I'm getting your point, but `cursor_value: "{{ last_records[-1]['key'] }}"` is mandatory for the pagination here.

The API returns the object ordered by creation date, and the pagination is possible by adding the last object key in the next request.

![Screenshot 2022-11-01 at 03 30 11](https://user-images.githubusercontent.com/65104164/199145677-ee50658b-9abe-4512-b37f-de0c6d34bdef.png)

Reply:
@elliottrabac I saw this connector supports incremental syncs. Are you interested in implementing this sync mode for this connector?

Reply:
Yes sure! Curious to see how it goes with the low code CDK 👍

I'll ping you once done.